src/styles/blocks/reset.scss

```diff
@@ -0,0 +1,35 @@
+body,
+ul {
+  margin: 0;
+}
+
+.nav {
+  &__list {
+    padding: 0;
+
+    list-style: none;
+  }
+
+  &__item {
+    &:first-child a {
+      margin-left: 0;
+    }
+
+    &:last-child a {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    text-decoration: none;
+  }
+}
+
+.price {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.reviews {
```

Comment:
The same
Actually, these classes shouldn't be separate BEM blocks, they are related to the `card`.
And BEM block couldn't have any margins